frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -157,7 +159,9 @@
             <div class="empty-category">{{ T.contestListEmpty }}</div>
           </div>
           <omegaup-contest-card
-            v-for="contestItem in contestList"
+            v-for="contestItem in currentOrder === ContestOrder.None
```

Comment:
```suggestion
            v-for="contestItem in sortedContestList
```

lo más conveniente es que en `sortedContestList` al principio se cheque si no hay que ordenar y ahí se regrese `this.contestList`. así todos estos `v-for` se simplifican.